web/src/beta/ui/fields/index.stories.tsx

```diff
@@ -0,0 +1,173 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import {
+  CommonField,
+  InputField,
+  TwinInputField,
+  TripletInputField,
+  QuartetInputField,
+  AssetField,
+  SelectField,
+  TextareaField,
+  SwitchField,
+  CodeField,
+  ColorField,
+  SpacingField,
+  CameraField,
+  SliderField,
+  TimePointField,
+  TimePeriodField,
+  ListField,
+  MultiSelectInputField,
+  TimeField,
+} from "./index";
+
+const meta: Meta<typeof InputField> = {
+  component: InputField,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof InputField>;
+
+const optionsMenu = [
+  {
+    id: "0",
+    title: "delete 1",
+    onClick: () => console.log("Option 1 deleted"),
+  },
+];
+
+export const Components: Story = {
+  render: () => (
+    <div style={{ display: "flex", flexDirection: "column", gap: "20px" }}>
+      <CommonField
+        commonTitle="Field Name"
+        description="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse varius enim in eros elementum tristique. Duis cursus, mi quis viverra ornare, eros dolor interdum nulla."
+      />
+      <InputField
+        placeholder="Text"
+        commonTitle="Field Name"
+        description="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse varius enim in eros elementum tristique. Duis cursus, mi quis viverra ornare, eros dolor interdum nulla."
+      />
+      <TwinInputField
+        values={["3.14159265", "4.14159265"]}
+        content={["Latitude", "Longitude"]}
+        commonTitle="Field Name"
+        description="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse varius enim in eros elementum tristique. Duis cursus, mi quis viverra ornare, eros dolor interdum nulla."
+      />
+      <TripletInputField
+        values={["3.14159265", "4.14159265", "3.14159265"]}
+        content={["Heading", "Pitch", "Roll"]}
```

Comment:
[consider the arrangement based on design](https://www.figma.com/design/W6nDSiCBLubcNeAT6RkOoa/%E2%AD%90%E2%AD%90%E2%AD%90Re%3AEarth-Beta-Design-system?node-id=5148-66171&t=XXiexPiXw0HCxbF2-0)

Reply:
done